### DISCUSSION TITLE
Range Columns

### DISCUSSION THREAD

**Discussion by sarkan-ag5** [go-jet/jet#305] [Ideas]
Hello there,

I wanted to ask if you have any plans to support PostgreSQL's range types. These types have been available for some time now and can be used with various data types, including integers, strings, timestamps, and dates. Alternatively, would you be open to accepting a pull request that adds support for these types? Thank you.

**Comment by go-jet** [comment]
Hi @sarkan-ag5 ,
Are you interested in adding range types support into SQL builder or to model types? 
If it is the latter, you can already use range types. Just [customize the generator](https://github.com/go-jet/jet/wiki/Generator#generator-customization) with your preferred range type implementation.

**Reply by sarkan-ag5** [reply]
Actually, I prefer having the fields on both sides so that we can accurately generate models that match the database schema. This will allow us to use methods like date range to generate SQL queries for these fields. I have already forked your repository and made a few changes. I will submit a pull request soon. I would like to hear your thoughts about these changes as well.
